packages/miniflare/src/plugins/core/index.ts

```diff
@@ -186,6 +186,8 @@ export const CoreSharedOptionsSchema = z.object({
 	httpsCertPath: z.string().optional(),
 
 	inspectorPort: z.number().optional(),
+	inspectorProxy: z.boolean().optional(),
```

Comment:
Should this maybe be `experimentalInspectorProxy` or something? 🤔

Reply:
Miniflare usually uses `unsafe` prefixes for things like that

Reply:
yeah I saw that but `unsafe` feels to me like something for things that should not generally be used but are potentially stable (like `unsafeEval`), but anyways if that's the pattern, and we do want to make this as not ready, I'm happy to go with `unsafeInspectorProxy` 🙂

Reply:
ok I've renamed this to `unsafeInspectorProxy` as I think it'd be safer to make this as experimental somehow (I'd also be happy to make this clearer with `experimentalInspectorProxy` although I'm usually a big fan of consistency 🙂 )